Cubical/Algebra/Module/Instances/FinMatrix.agda

```diff
@@ -0,0 +1,39 @@
+{-# OPTIONS --safe #-}
+module Cubical.Algebra.Module.Instances.FinMatrix where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.SIP
```

Comment:
Do you use that somewhere? If you just use the angel-brackets, you can 
```agda
open import Cubical.Foundations.Structure using (⟨_⟩)
```

Reply:
I am using for `str` too, so I made this change.